templates/base.html

```diff
@@ -97,6 +97,8 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
     <!-- Include all compiled plugins (below), or include individual files as needed -->
     <script src="{% static 'js/bootstrap.min.js' %}"></script>
+    {% block additional_js %}
```

Comment:
Django CMS comes with sekizai tags, so one could use them for the additional js. It provides some advantages like removal of duplicated includes, adding is from multiple sub templates without simply overriding it, adding post processors etc.

Reply:
To be honest, I saw that _render_block_ and started googling it, but couldn't force it to work. So being lazy I've used classic blocks, but I'll look into it, because it seems useful